src/routes/Login/Login.js

```diff
@@ -27,38 +28,51 @@ const Login = () => {
   console.log(errors)
 
   return (
-    <Column as='form' onSubmit={handleSubmit(onSubmit)} p={40} alignItems='center'>
-      <Input name='name' register={register} label='Nome' placeholder='Seu Nome' error={errors.name?.message} />
+    <Column as='form' onSubmit={handleSubmit(onSubmit)} p={40} alignItems='center' backgroundColor='#343a50'>
```

Comment:
Esse backgroundColor escuro só está na volta do Form, e por isso tá ficando uma barra branca abaixo do form. Era esse o comportamento esperado? Caso contrário, se a ideia for adicionar essa cor para o background das páginas, recomendaria adicionar esse fundo no createGlobalStyle com o selector do body. O que acha?